src/main/java/VASSAL/tools/image/GeneralFilter.java

```diff
@@ -106,7 +106,9 @@ public float apply(float t) {
     }
 
     @Override
-    public float getSamplingRadius() { return 1.0f; }
+    public float getSamplingRadius() {
```

Comment:
Very short function bodies should be able to be single-line.

Reply:
The general rule is to avoid having several statements on a single line. Adding exceptions to formatting rules makes the rules too complex, in this particular case it adds extra work to further edits of the code (what if I want to add a log statement during debugging?), makes setting breakpoints ambigious (is the breakpoint on the whole method or on the return statement?). Also, a good IDE will display the expanded version in a single line.